sort-table.js

```diff
@@ -0,0 +1,31 @@
+const grid = document.getElementById('grid');
+
+grid.onclick = function(e) {
```

Comment:
use addEventListener instead
do not use one letter variables, `event` is more explicite